concrete/config/concrete.php

```diff
@@ -6,9 +6,9 @@
      *
      * @var string
      */
-    'version' => '8.5.0a3',
-    'version_installed' => '8.5.0a3',
-    'version_db' => '20181112211702', // the key of the latest database migration
+    'version' => '8.5.0a4',
+    'version_installed' => '8.5.0a4',
+    'version_db' => 'Version20181204000000', // the key of the latest database migration
```

Comment:
Here we need just the version number

Reply:
Reverted in bc1eb34fe08359247eb0f1be568fdfee86db75c7

Reply:
I'd let @aembler manage the `version` configuration key, but I'd set the `version_db` key in the PR (to `'20181204000000'` instead of `'Version20181204000000'`)